client/src/Components/Button/index.tsx

```diff
@@ -25,26 +27,26 @@ export interface ButtonProps {
   onClick?: () => void
 }
 
-/**
- * Primary UI component for user interaction
- */
 export const Button: React.FC<ButtonProps> = ({
-  primary = false,
-  size = 'medium',
-  backgroundColor,
+  backgroundColour = 'primary',
+  textColour = 'white',
   label,
   ...props
 }) => {
-  const mode = primary
-    ? 'storybook-button--primary'
-    : 'storybook-button--secondary'
   return (
     <button
       type='button'
-      className={['storybook-button', `storybook-button--${size}`, mode].join(
-        ' '
-      )}
-      style={{ backgroundColor }}
+      className={[
+        'font-serif',
+        `text-${textColour}`,
+        `shadow-${backgroundColour}`,
+        `bg-${backgroundColour}`,
```

Comment:
tailwind does not work when using string templates as such. Have a look at how I did the textfield component when a prop has a conditional